UPGRADE-2.0.md

```diff
@@ -1,5 +1,38 @@
 # Upgrade from 1.10 to 2.0
 
+Version `2.x` is a complete rewrite of the existing code base. The public API has been trimmed down to a minimum.
+The preferred way of using the SDK is through our static API.
+
+Here is a simple example to get started:
+
+```php
+\Sentry\init(['dsn' => '___PUBLIC_DSN___' ]);
+
+try {
```

Comment:
I would move the try block below the configure scope and breadcrumb to make it more clear those should become before exceptions are being thrown.